src/python/bot/fuzzers/libFuzzer/engine.py

```diff
@@ -0,0 +1,58 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""libFuzzer engine interface."""
+
+from bot.fuzzers import engine
+
+
+class LibFuzzerOptions(engine.FuzzOptions):
```

Comment:
nit: why not just `Options`? For instance, `Engine` is not `FuzzEngine` :)

Reply:
Because these options are specific to fuzzing :)